src/en/draft-benchmarking.md

```diff
@@ -0,0 +1,147 @@
+# Why benchmarking
```

Comment:
Could you put in a metadata title as the first line please:

Title: Adding benchmarks to your charm